storage/chunker_test.go

```diff
@@ -238,7 +238,7 @@ func TestRandomData(t *testing.T) {
 	// This test can validate files up to a relatively short length, as tree chunker slows down drastically.
 	// Validation of longer files is done by TestLocalStoreAndRetrieve in swarm package.
 	//sizes := []int{1, 60, 83, 179, 253, 1024, 4095, 4096, 4097, 8191, 8192, 8193, 12287, 12288, 12289, 524288, 524288 + 1, 524288 + 4097, 7 * 524288, 7*524288 + 1, 7*524288 + 4097}
-	sizes := []int{1, 60, 83, 179, 253, 1024, 4095, 4097, 8191, 8192, 12288, 12289, 524288}
+	sizes := []int{1, 60, 83, 179, 253, 1024, 4095, 4097, 8191, 8192, 12288, 12289, 524288, 2345678}
```

Comment:
This is also passing fine on `master`, not sure why we are adding another test case?

Reply:
It does not affect the old code.This was added here to catch bugs in this PR.. We have 150 storage threads now...This size 2345678 will exhaust the 150 storage go-routines.. and if any one of them is not release the properly.. the test case will fails.

Reply:
OK, fair enough, it doesn't make the test suite much slower on Travis, so its fine.

@jmozah please add it in the commented line above as well - we don't run all these tests, because Travis is slow, but it is expected that if you touch this code, you run them manually locally.

This code is not changed often, so we decided this is an OK compromise to make.